scripts/linux/buildLocalEdgeHub.sh

```diff
@@ -55,6 +55,14 @@ function process_args() {
 
 process_args "$@"
 
-scripts/linux/buildBranch.sh --no-rocksdb-bin
-scripts/linux/buildRocksDb.sh --output-dir "$(pwd)/target/publish/Microsoft.Azure.Devices.Edge.Hub.Service" --build-number debug --arch amd64
-scripts/linux/buildImage.sh -r "$REGISTRY_ADDRESS" -i azureiotedge-hub -n microsoft -P Microsoft.Azure.Devices.Edge.Hub.Service -v "$VERSION" -t amd64 --bin-dir target
+scripts/linux/buildBranch.sh --no-rocksdb-bin --skip-quickstart
+scripts/linux/buildRocksDb.sh \
+    --output-dir "$(pwd)/target/publish/Microsoft.Azure.Devices.Edge.Hub.Service" \
+    --build-number debug \
+    --arch amd64
+docker build \
+    --file "$(pwd)/target/publish/Microsoft.Azure.Devices.Edge.Hub.Service/docker/linux/Dockerfile" \
+    --tag "$REGISTRY_ADDRESS/microsoft/azureiotedge-hub:$VERSION-linux-amd64" \
+    --build-arg EXE_DIR=. \
+    --no-cache \
```

Comment:
This should work in the pipeline that we are running. Though it will have a side effect on `Build Edge Hub Container Locally` section of the devguide.md

Reply:
I believe I tested this locally. If you call `buildLocalEdgeHub.sh` from your local command line it should work.

Reply:
Ok, I ran buildLocalEdgeHub.sh locally, and there were some updates I needed to make. The script was still using docker's built-in build machinery, which doesn't understand the $TARGETPLATFORM env var in the Dockerfiles. So I updated it to use `docker buildx build`, similar to what buildImage.sh does. I confirmed that the script now works the way it used to.